lib/templates/json_texture.template.js

```diff
@@ -0,0 +1,43 @@
+path = require("path");
+
+function jsonTextureTemplate(data) {
+  var spriteObj = {};
+
+  // Create frame data for each sprite.
+  spriteObj.frames={};
```

Comment:
Can we please fix up the spacing here to be consistent with the rest of the repo?

```js
spriteObj.frames = {};
```